content/doc/developer/tutorial-improve/_create-a-branch.adoc

```diff
@@ -7,7 +7,7 @@
 
 == Create a branch
 
-Create a git branch for your local work with the command:
+In a local copy of your link:https://docs.github.com/en/get-started/quickstart/fork-a-repo[fork] of the plugin repository create a git branch for your work with the command:
```

Comment:
```suggestion
In a local copy of your link:https://docs.github.com/en/get-started/quickstart/fork-a-repo[fork] of the plugin repository create a link:https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository[git branch] for your work with the command:
```
In case this is unclear, thoughts?